metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/config/Neo4jConfig.java

```diff
@@ -0,0 +1,35 @@
+package com.linkedin.metadata.kafka.config;
+
+import org.neo4j.driver.v1.Driver;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import com.linkedin.common.factory.Neo4jDriverFactory;
+import com.linkedin.metadata.dao.internal.BaseGraphWriterDAO;
+import com.linkedin.metadata.dao.internal.Neo4jGraphWriterDAO;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Configuration
+@Import({Neo4jDriverFactory.class})
+@RequiredArgsConstructor
+public class Neo4jConfig {
+
+    private final Driver neo4jDriver;
+
+    @Bean
+     public BaseGraphWriterDAO graphWriterDAO() {
+        BaseGraphWriterDAO graphWriterDAO = null;
+        try {
+            graphWriterDAO = new Neo4jGraphWriterDAO(neo4jDriver);
+        } catch (Exception e) {
+            log.error("Error in initializing Neo4j.", e);
```

Comment:
Throw a RuntimeException maybe? It just prints an error message and continue to execute. In the case of an exception, you will see an error message as below:
```
Error in initializing Neo4j.
Neo4jDriver built successfully
```